.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "spras"]
+	path = spras
+	url = https://github.com/Reed-CompBio/spras
```

Comment:
This is another chicken-and-egg problem. should we require a specific tag for SPRAS here, or should we have a Github Action within SPRAS that checks that any major PR makes this stable?

Reply:
The git submodule is currently pinned at a specific commit ([61f566](https://github.com/Reed-CompBio/spras/tree/61f5667f8cc1dfc156e2e388b75d381e714f91cc)) and needs to be updated - SPRAS tags would be a nicer idea, though.